app/helpers/entities_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module EntitiesHelper
+  def entity_name(entity)
+    entity.company_name || full_name(entity)
+  end
```

Comment:
I don't think this method needed

use decorator instead

`IdentityDecorator` and `AddressDecorator` already contains `display_name` method